frontend/device/pocketbook/device.lua

```diff
@@ -702,6 +702,24 @@ function PocketBook743C._fb_init(fb, finfo, vinfo)
     vinfo.bits_per_pixel = 24
 end
 
+-- PocketBook InkPad Color 3 (743K3)
+local PocketBook743K3 = PocketBook:extend{
+    model = "PBInkPadColor3",
+    display_dpi = 300,
+    color_saturation = 1.5,
+    hasColorScreen = yes,
+    canHWDither = yes, -- Adjust color saturation with inkview
+    canUseCBB = no, -- 24bpp
+    isAlwaysPortrait = yes,
+    usingForcedRotation = landscape_ccw,
+    hasNaturalLight = yes,
```

Comment:
Just to check. The original inkpads didn't have natural light because of the color screen. Did this change with the new model?

Reply:
PocketBook Color and PocketBook InkPad Color did not yet have additional orange LEDs. However, PocketBook Color 2 and PocketBook Color 3 have these.